cdp-agentkit-core/typescript/src/actions/cdp/data/pyth/fetch_price_feed_id.ts

```diff
@@ -0,0 +1,56 @@
+import { CdpAction } from "../../cdp_action";
+import { z } from "zod";
+
+const PYTH_FETCH_PRICE_FEED_ID_PROMPT = `
+Fetch the price feed ID for a given token symbol from Pyth.
+`;
+
+/**
+ * Input schema for Pyth fetch price feed ID action.
+ */
+export const PythFetchPriceFeedIDInput = z.object({
+  tokenSymbol: z.string().describe("The token symbol to fetch the price feed ID for"),
```

Comment:
What happens if someone inputs not a symbol but a full name (e.g. "Fetch the price of Bitcoin")?

Reply:
The agent is able to map from asset name to ticker symbol for common assets. It asks for a ticker otherwise.